besca/tl/auto_annot/_auto_annot.py

```diff
@@ -641,191 +639,6 @@ def predict(classifier, scaler, adata_pred, threshold = 0):
     return results 
 
 
-def report(adata_pred, celltype, method, analysis_name, train_datasets, test_dataset, merge, use_raw = False, genes_to_use = 'all', remove_nonshared = False, clustering = 'leiden', asymmetric_matrix = True):
```

Comment:
Instead of removing it, I would make it calling the new function and comment that it is DEPRECATED, to ensure backwards-combability with older auto-annot notebooks:
```
def report(adata_pred, celltype, method, analysis_name, train_datasets, test_dataset, merge, use_raw = False, genes_to_use = 'all', remove_nonshared = False, clustering = 'leiden', asymmetric_matrix = True):
    tl.report(adata_pred, celltype, method, analysis_name, train_datasets, test_dataset, merge, use_raw, genes_to_use, remove_nonshared, clustering, asymmetric_matrix)
```

Reply:
Yes seems good. I ll update the PR.